src/bridge/cockpitconnect.c

```diff
@@ -220,6 +220,13 @@ parse_address (CockpitChannel *channel,
       goto out;
     }
 
+  if (unix_path && json_object_has_member (options, "tls"))
```

Comment:
can we think of a way to do this without modifying the C?

Reply:
We could if we could make JS unit tests specific to the bridge. My current idea: add some /info endpoint to test-server which gives you a JSON object like `{ "slow_tests": true, "bridge": "cockpit-bridge.pyz" }`. Then unit tests can query that and skip tests under valgrind or conditionalize expectations on the bridge. This would also allow us to re-enable the barrier test which you disabled when landing the pybridge.

Reply:
This is perfect.

Reply:
I implemented that. But I can't land it separately, as querying the /mock/info path requires... *drumroll* cockpit.http() :grin: So I'll land it all in #17631.